src/utility/fetch-capture-model.ts

```diff
@@ -0,0 +1,17 @@
+import { CaptureModel } from '../types/capture-model';
+
+export async function fetchCaptureModel(
+  captureModelId?: string,
+  captureModel?: CaptureModel
+): Promise<CaptureModel> {
+  if (captureModel) {
+    return captureModel;
+  }
+
+  if (captureModelId) {
+    // @todo validation of incoming JSON.
+    return (await (await fetch(captureModelId)).json()) as CaptureModel;
```

Comment:
Is fetch polyfilled by the FESK webpack config?

Reply:
not at the moment, no. Not completely decided on wether this repository will be a library with all the pieces to make an application, or include an application itself. My thinking is that the app that uses all of these components will provide the polyfill. Good spot though, I didn't think about that